test/addons/rbd-mirror/start-data/vgrc-sample.yaml

```diff
@@ -6,9 +6,14 @@ apiVersion: replication.storage.openshift.io/v1alpha1
 kind: VolumeGroupReplicationClass
 metadata:
   name: vgrc-sample
+  labels:
+    ramendr.openshift.io/storageid: rook-ceph-$cluster-1
+    ramendr.openshift.io/replicationid: rook-ceph-replication-1
 spec:
   provisioner: rook-ceph.rbd.csi.ceph.com
   parameters:
-    replication.storage.openshift.io/replication-secret-name: rook-csi-rbd-provisioner
-    replication.storage.openshift.io/replication-secret-namespace: rook-ceph
+    clusterID: rook-ceph
```

Comment:
Can you explain what this `clusterID` means. There is also a `ClusterId` in PeerClass struct, the id is retrieved from kube-system of the cluster, check [here.](https://github.com/RamenDR/ramen/blob/main/api/v1alpha1/drpolicy_types.go#L103)

Reply:
@rakeshgm As far as I understood from Nikhil's explanation, it is the id of ceph cluster. It is mandatory in volumegroupreplicationclass. [here](https://github.com/csi-addons/kubernetes-csi-addons/blob/60f22e15eb654d3700aecf5d2900c080ad68c853/cmd/csi-addons/replication.go#L45)

Reply:
This is not a ceph cluster ID, this is the clusterID which is used to get the ceph mon and other details. we have same in storageclass and volumegroupsnapshot class as well.